giphy-search/src/adapters/handleFetch.js

```diff
@@ -0,0 +1,32 @@
+export const handleFetch = async (url, options = {}) => {
```

Comment:
Just moved this from a `utils` folder to the `adapter` folder where it will exclusively be used.